components/Movie/index.js

```diff
@@ -1,43 +1,99 @@
 import React from "react";
-import { useContext } from "react";
-import { DataContext } from "../../pages/_app";
 import Image from "next/image";
 import styled from "styled-components";
+import genres from "../../pages/api/genres";
+import { useState, useEffect } from "react";
 
 const StyledDiv = styled.div`
   height: 180px;
-  width: 325px;
-  background-color: lightgray;
+  width: 130px;
 `;
 
-/* The lightgray background is temporarily used for display purposes.
-Will be changed in the future. */
-
 const StyledSection = styled.section`
   display: flex;
   justify-content: center;
   margin-bottom: 20px;
 `;
 
-export default function Movie() {
-  const { movies } = useContext(DataContext);
+export default function Movie({ movie }) {
+  const [runtime, setRuntime] = useState(0);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await fetch(
+          `https://api.themoviedb.org/3/movie/${movie.id}?api_key=657b6c5e2a2ab2cafa267e54252ca1a7&language=eng-US`
```

Comment:
You just committed your `API` key here. Highly recommend put it in an `.env` file.

Better: Delete this api key from your account, create a new one, and put this new one in the `.env` file.